docs/user/kafka/confluent.rst

```diff
@@ -163,3 +163,62 @@ Example GeoMesa Avro Schema
         }
       ]
     }
+
+Schema Overrides Config
```

Comment:
@elahrvivaz any other places that things like `kafka.schema.overrides` needs to be documented?

Reply:
i think this is the only docs page that deals with confluent, so i don't think it needs to be anywhere else